server/src/profile/api.rs

```diff
@@ -289,6 +295,7 @@ impl InitData {
         };
 
         Ok(InitData {
+            analysis: AnalysisService::new_sized(run.analysis.max_cache_size),
```

Comment:
I think it would be great it one could construct an `AnalysisService` from an `AnalysisConfig` directly.

Reply:
Will change that around. And give AnalysisConfig a default.